satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -348,163 +417,147 @@ datasets:
     layer: 1
     standard_name: air_pressure_at_cloud_top
 
-  retrieved_cloud_top_height:
-    name: retrieved_cloud_top_height
-    resolution: 2000
-    file_type: nc_fci_oca
-    file_key: retrieved_cloud_top_height
-    standard_name: height_at_cloud_top
-
-  retrieval_error_cloud_optical_thickness_upper_layer:
-    name: retrieval_error_cloud_optical_thickness_upper_layer
-    resolution: 2000
-    file_type: nc_fci_oca
-    file_key: retrieval_error_cloud_optical_thickness
-    layer: 0
-    long_name: cloud_optical_depth
-
-  retrieval_error_cloud_optical_thickness_lower_layer:
-    name: retrieval_error_cloud_optical_thickness_lower_layer
+  retrieval_error_cloud_top_pressure_lower_layer:
+    name: retrieval_error_cloud_top_pressure_lower_layer
     resolution: 2000
     file_type: nc_fci_oca
-    file_key: retrieval_error_cloud_optical_thickness
+    file_key: retrieval_error_cloud_top_pressure
     layer: 1
-    long_name: cloud_optical_depth
+    standard_name: air_pressure_at_cloud_top standard_erro
```

Comment:
Here I would add the `long_name` from the FCIL2FS but with the addition `"(lower layer)"` at the end

Reply:
appended "for upper/lower layer" to the long names